### DISCUSSION TITLE
cant download any package using npm

### DISCUSSION THREAD

**Discussion by m0x61h0x64i** [community/community#36250] [Packages] [answered]
hello, i cant install ant package using npm, my npm version : 
```
8.15.0
```

what should i do? is there a way to reset vscode terminal network or any fix is available ?

**Comment by ihnatenkoo** [comment, marked as answer]
Hello, do you try clean cash in your npm?
If not, run the `npm cache clean --force command in your terminal`

And you can try update you npn version:
`npm i npm`

**Reply by m0x61h0x64i** [reply]
> Hello, do you try clean cash in your npm? If not, run the `npm cache clean --force command in your terminal`
> 
> And you can try update you npn version: `npm i npm`

i did run that command, but while updating npm it still gives me error:
```
$ npm i npm

npm ERR! code ETIMEDOUT
npm ERR! syscall connect
npm ERR! errno ETIMEDOUT
npm ERR! network request to https://registry.npmjs.org/npm failed, reason: connect ETIMEDOUT 148.251.160.244:443
npm ERR! network This is a problem related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network 
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\user\AppData\Local\npm-cache\_logs\2022-10-15T16_11_43_326Z-debug-0.log
```

**Reply by ihnatenkoo** [reply]
Do you have .nmprc file ? it locates on the paths C:\Users\.nmprc

**Reply by m0x61h0x64i** [reply]
> Do you have .nmprc file ? it locates on the paths C:\Users.nmprc

no.
![image](https://user-images.githubusercontent.com/109550653/195999557-3bd7f459-d043-4fd1-b37f-ff49cc38833b.png)

**Reply by ihnatenkoo** [reply]
Ok, try to delete wrong proxy settings using command
`npm config delete proxy`

**Reply by m0x61h0x64i** [reply]
> Ok, try to delete wrong proxy settings using command `npm config delete proxy`

thanks. now it works!

**Comment by ossanna16** [comment]
Hey there @m0x31h0x64i and thank you for asking a good question! I'm glad you found our community 🙂
